PyExpLabSys/drivers/keithley_2400.py

```diff
@@ -110,14 +110,17 @@ def read_current(self):
         # Only the current is measured, voltage is either
         # NaN or the source-setpoint.
         values = raw.split(',')
+        print('DEBUG KLAF 2400 CURRENT', values)
```

Comment:
If we want to keep the debug messages later on, they could be toggled on and off via a self.debug attribute